apps/ui/src/networks/index.ts

```diff
@@ -65,7 +67,7 @@ export const explorePageProtocols: Record<ExplorePageProtocol, ProtocolConfig> =
   snapshot: {
     key: 'snapshot',
     label: 'Snapshot',
-    networks: enabledNetworks.filter(network => offchainNetworks.includes(network)),
+    networks: [metadataNetwork],
```

Comment:
Isn't one of this PR goal to also allow showing both `s` and `s-tn` data ?

Reply:
You can read both, just explore page displays one or another as it was mentioned we want one or another there.
You can access any space via direct URLs.